### ISSUE TITLE
missing images in W3CN1302 / I3CN1308

### ISSUE THREAD

**Issue by eroux** [buda-base/library-issues#300]
Volume 5 (I3CN1308) of W3CN1302 is strangely lacking some images. I've asked Bruno for their scans and he sent me what he has: http://eroux.fr/tsarang/05.zip . The first thing to note is that these images are in black and white while ours are in color... I'm not sure how we got the color images? The main problem is that there are many more black and white images in this volume than there are color images (52 one-side color vs 352 two-sides black and white). So I think we have two solutions, we could either:
- create a new image group with these images
- add these new images in the same image group, with two options:
   * not having duplicates (not adding the black and white images  that correspond to color images we already have)
   * add everything even what's already there

I'm personally favorable to the last option (with duplicates). @karmagongde what do you think? Do you know why we have black and white images for other volumes but not this one?

Also, I'm downloading the images of all the volumes so that we can at least have them in our src/ directory, I don't think we currently do.

**Comment by TBRC-Travis** [comment]
@eroux would it be possible to get color images of the missing pages?  if B&W is the best we can do then I would vote for adding them to the same image group.  I don't like the idea of having duplicates as it gets a little confusing in terms of image management but if BUDA has a mechanism for hiding duplicates from view then technically I suppose it's okay to have them around.

@karmagongde when these are ready for syncing please add them to `ReadyToProcess/Replacement`

**Comment by eroux** [comment]
rKTs does seem to only have B&W images, so AFAIK we can't get the color images for the rest... but parts of the story are still unclear (like: where did we get the color images from?) so maybe that will change.

We have a mechanism to hide duplicates in BUDA yes, that won't be a problem.

**Comment by brunogml** [comment]
Please just wait a couple of days. I am sure that the B&W and the color images are mine. I must check what happend here. It will take some days before I'm gone through all my old drives. Thank for being patient.

**Comment by karmagongde** [comment]
@brunogml Thank you for looking the both images from your backup storage. I can't find beside the online posted. According to the the catalog record we received both from you.
@eroux Elie, I am not able download the images you have posted online.  http://eroux.fr/tsarang/completebnw.zip
It said, completebnw.zip can't be downloaded securely.

**Comment by eroux** [comment]
@karmagongde I've unzipped the images in `Incoming/Inbox/scans/Inbox/W3CN1302` on RS5

**Comment by karmagongde** [comment]
@eroux Thank Elie, I checked the images you have uploaded and the back ground is black and scripts are white. I hope Bruno have a better images.

**Comment by eroux** [comment]
I'll let Bruno answer in more details if he has time, but in short, the filming was made on the side of the NGMPP project with some leftover materials, and there aren't color images for everything... so there's no color images for this volume outside of those we already have. @karmagongde or @TBRC-Travis can you copy the black and white images (for all volumes) to some `source/`directory (I don't think we have the originals in our archives), and also complete volume 5 with the black and white images? (they'll need to be cropped so it will take a bit of time... also for the web images you don't have to put the duplicates of those we already have in color I think) It's one of the volumes that has a text that doesn't appear in other Kangyurs so it's quite interesting!

**Comment by TBRC-Travis** [comment]
@eroux ok, good to know.  sounds like we'll just move forward with the b&w images from Bruno for this volume and reconcile anything that had color images available.  in terms of the "sources" directory for this work there's already one in the archive but I'll take a closer look and sort it out.

**Comment by eroux** [comment]
perfect thanks! It's interesting, I actually took a look at the archives the other day to check the source directory but couldn't find it... I guess I didn't look correctly, which path does it have?

**Comment by TBRC-Travis** [comment]
where were you looking?

**Comment by eroux** [comment]
sorry, just found it, forget about it! Not sure what I did the last time...

**Comment by TBRC-Travis** [comment]
great

**Comment by eroux** [comment]
@TBRC-Travis @karmagongde what are the next steps for this? Travis, are you ready to process the images or should Karma do some additional manipulation?

**Comment by TBRC-Travis** [comment]
@eroux I mistakenly thought we were waiting on images here.  looking back through the thread it seems we have everything we need.  I'll take a look at this later today.

**Comment by eroux** [comment]
excellent, thanks!

**Comment by TBRC-Travis** [comment]
@eroux I sent the B&W images over to Tim for cropping.  We'll have this sorted out sometime next week.

**Comment by eroux** [comment]
ok, thanks for the update!

**Comment by TBRC-Travis** [comment]
@eroux can you check the images in 09-rgyud-Ta-(Tha)?  It seems that 067.tif through 081.tif are corrupted and wondering if you have good copies of these or not.

**Comment by eroux** [comment]
ah indeed, thanks for spotting that! I've asked Bruno, let's see... that shouldn't impact the volume to crop though right? It's volume 5 (or are there images missing in volume 9 too?)

**Comment by TBRC-Travis** [comment]
ok great.  let's see what Bruno comes back with.  it doesn't impact volume 5 but we're cropping all the volumes just in case we need to rely on the B&W images for whatever reason they'll be ready to go.

**Comment by TBRC-Travis** [comment]
@eroux I've synced the B&W images for the missing pages of Volume 5 (I3CN1308), though for some reason I can't get the viewer to load images for this work now:  https://library.bdrc.io/show/bdr:W3CN1302

also, though the other volumes have color images available I noticed the B&W images are higher resolution.  I'm not sure whether or not we want to replace the color images with the B&W versions but that's also an option.

**Comment by eroux** [comment]
thanks! Interesting problem, it's because I created a custom manifest with the old images (to add the pagination) but it only contains the previous images, I'll update it tomorrow

**Comment by eroux** [comment]
also, interesting consideration about the B&W images... I don't really know what's best... let's keep it that way, I'll ask Bruno if he has an opinion (his team is using the images, he'll know what's more readable). In any case, closing this issue, thanks again!
